project/src/test/java/com/google/sps/VolunteerOpportunityTest.java

```diff
@@ -0,0 +1,100 @@
+package com.google.sps;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.sps.data.VolunteeringOpportunity;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class VolunteerOpportunityTest {
+  private static final String NAME = "Performer";
+  private static final int NUMBER_OF_SPOTS = 240;
+  private static final String PLAYING_AN_INSTRUMENT = "Playing an instrument";
+  private static final String PERFORMING = "Performing";
+  private static final String DANCING = "Dancing";
+  private static final Set<String> SKILLS = ImmutableSet.of(PLAYING_AN_INSTRUMENT, PERFORMING);
+  private static final Set<String> SKILLS_WITH_DANCING =
+      ImmutableSet.of(PLAYING_AN_INSTRUMENT, PERFORMING, DANCING);
+
+  @Test
```

Comment:
can you comment what the test does for clarity

Reply:
I dont think its necessary because the name has the purpose of the test.